pkgs/development/python-modules/gurobipy/linux.nix

```
@@ -1,21 +1,19 @@
 { fetchurl, python }:
-assert python.pkgs.isPy27;
+assert with python.pkgs; isPy27 || isPy36 || isPy37 || isPy38 || isPy39;
 
 python.pkgs.buildPythonPackage
   { pname = "gurobipy";
-    version = "7.5.2";
+    version = "9.1.2";
     src = fetchurl
-      { url = "http://packages.gurobi.com/7.5/gurobi7.5.2_linux64.tar.gz";
-        sha256 = "13i1dl22lnmg7z9mb48zl3hy1qnpwdpr0zl2aizda0qnb7my5rnj";
+      { url = "http://packages.gurobi.com/9.1/gurobi9.1.2_linux64.tar.gz";
```

```suggestion
      { url = "https://packages.gurobi.com/9.1/gurobi9.1.2_linux64.tar.gz";
```

Please substitute the version.